src/main/java/mate/academy/internetshop/controller/DeleteProductFromShoppingCartController.java

```diff
@@ -13,6 +13,7 @@
 
 @WebServlet("/shoppingCart/delete")
```

Comment:
```suggestion
@WebServlet("/shoppingCart/products/delete")
```
щоб уточнити, що йде не видалення корзини а видалення продукту з корзини